sdk/identity/azure-identity-extensions/Azure-Database-for-PostgreSQL-README.md

```diff
@@ -114,10 +114,10 @@ az postgres server firewall-rule create \
     --output tsv
 ```
 
-#### Enable Azure AD-based authentication
+#### Enable Microsoft Entra ID-based authentication
```

Comment:
For example here - just below you drop `ID`. This is probably fine, just want to make sure.